addons/a11y/src/manager.tsx

```diff
@@ -3,9 +3,9 @@ import { addons, types } from '@storybook/addons';
 import { ADDON_ID, PANEL_ID, PARAM_KEY } from './constants';
 import { VisionSimulator } from './components/VisionSimulator';
 import { A11YPanel } from './components/A11YPanel';
-import { A11yContextProvider } from './components/A11yContext';
+import { A11yContextProvider, Results } from './components/A11yContext';
```

Comment:
Hey maintainers, I have a DeepScan issue on this line
`Imported binding 'Results' is not used.`
It is used in line 18 for typing. Is it a false positive, or is it because the scan is done on compiled code that removes typings ? 
I don't know how to handle this one